src/library/Connect/WorkloadInformationProvider.cs

```diff
@@ -117,6 +120,28 @@ internal class WorkloadInformationProvider : IWorkloadInformationProvider
             }
         }
 
+        //
+        public List<int> GetPortsToIgnoreFromAnnotations(V1ServiceList serviceNameSpace) {
+            var portsToIgnore = new List<int>();
+            foreach(var item in serviceNameSpace.Items) {
+                if (item.Metadata.Annotations != null) {
+                    if (item.Metadata.Annotations.TryGetValue("bridgetokubernetes/ignore-ports", out var ports)) {
+                        var allPorts = ports.Split(",");
+                        foreach(var port in allPorts) {
+                            try {
+                                portsToIgnore.Add(int.Parse(port));
+                            } catch {
+                                throw new Exception("The ports are not in the correct format");
+                            }
+                            
+                        }
+                    }
+                }
+            }
+            _log.Info("Ignoring ports: {0}", portsToIgnore.Count > 0 ? portsToIgnore.ToString() : "none");

```

Comment:
Maybe something more explicit on this log. For example: 

"Per bridgetokubernetes/ignore-ports user config the following ports will be ignored: {0}"